src/strategies/ethermon-erc721/examples.json

```diff
@@ -0,0 +1,1484 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "ethermon-erc721",
+      "params": {
+        "EMON_DATA_ADDRESS": "0x5d00d312e171be5342067c09bae883f9bcb2003b",
+        "EMONA_ADDRESS": "0xabc1c404424bdf24c19a5cc5ef8f47781d18eb3e",
+        "symbol": "EMONA",
+        "classIdWeight": {
```

Comment:
It is not good idea to store all these weights inside strategy params, maybe you can store these in a IPFS file and fetch it when required?